qtpy/tests/test_qtprintsupport.py

```diff
@@ -12,5 +17,22 @@ def test_qtprintsupport():
     assert QtPrintSupport.QPrinter is not None
     assert QtPrintSupport.QPrinterInfo is not None
     assert QtPrintSupport.QPrintPreviewWidget is not None
-	
 
+
+def test_qpagesetupdialog_exec_():
+    """Test qtpy.QtPrintSupport.QPageSetupDialog exec_"""
+    assert QtPrintSupport.QPageSetupDialog.exec_ is not None
+
+
+def test_qprintdialog_exec_():
+    """Test qtpy.QtPrintSupport.QPrintDialog exec_"""
+    assert QtPrintSupport.QPrintDialog.exec_ is not None
+
+
+@pytest.mark.skipif(sys.platform.startswith('linux') and not_using_conda,
+                    reason="Fatal Python error: Aborted on Linux CI when not using conda")
```

Comment:
```suggestion
@pytest.mark.skipif(
    sys.platform.startswith('linux') and not_using_conda,
    reason="Fatal Python error: Aborted on Linux CI when not using conda")
```

Reply:
```suggestion
@pytest.mark.skipif(
    sys.platform.startswith('linux') and not_using_conda(),
    reason="Fatal Python error: Aborted on Linux CI when not using conda")
```

Fix syntax error with the above suggestion